src/extras-openmoji/climate-environment/E2CF.svg

```diff
@@ -0,0 +1,32 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```

Comment:
would be good to connect the car and the fumes but not important to change

Reply:
![2020-02-27 at 12 18](https://user-images.githubusercontent.com/480224/75440063-4ef81a80-595b-11ea-973b-80967f8dbf3c.jpg)